l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -343,7 +343,13 @@ def setCedentePrestatore(self, company, fatturapa):
     def _setDatiAnagraficiCessionario(self, partner, fatturapa):
         fatturapa.FatturaElettronicaHeader.CessionarioCommittente.\
             DatiAnagrafici = DatiAnagraficiCessionarioType()
-        if not partner.vat and not partner.fiscalcode:
+        if not partner.vat and not partner.fiscalcode and partner.codice_destinatario == 'XXXXXXX':
+            # SDI accpets missing VAT# for foreign customers by setting a fake IdCodice and a valid IdPaese
+            # Otherwise raise error if we havo no VAT# and no Fiscal code
+            fatturapa.FatturaElettronicaHeader.CessionarioCommittente.\
+                DatiAnagrafici.IdFiscaleIVA = IdFiscaleType(
+                    IdPaese=partner.country_id.code, IdCodice='99999999999')
+        if not partner.vat and not partner.fiscalcode and not partner.codice_destinatario == 'XXXXXXX': 
```

Comment:
I think this check should be also changed in `l10n_it_fatturapa/models/partner.py`: `_check_ftpa_partner_data`

Reply:
You are right, it is at line 81. And we also should not check Provincia and CAP for foreign partners.